pkg/apis/operator/v1alpha1/tektonaddon_defaults.go

```diff
@@ -54,8 +52,7 @@ func setAddonDefaults(addon *Addon) {
 		}
 	}
 
-	// by default enable pac
-	if addon.EnablePAC == nil {
-		addon.EnablePAC = ptr.Bool(false)
-	}
+	// Deprecated, will be removed in further releases
+	// moved to OpenShift platform section
+	addon.EnablePAC = nil
```

Comment:
Could you specify that in doc here https://github.com/tektoncd/operator/blob/main/docs/TektonAddon.md

Reply:
https://github.com/tektoncd/operator/issues/1138